components/Article.jsx

```diff
@@ -0,0 +1,38 @@
+import {Link} from "react-router-dom";
+
+export const Article = (article) => {
+    const {author, title, article_id, topic, created_at, votes, article_img_url, comment_count, body} = article.article;
+
+    const formatDate = (dateString) => {
+        const options = {year: "numeric", month: "long", day: "numeric"};
+        return new Date(dateString).toLocaleDateString(undefined, options);
+    };
+
+    return (
+        <article className="flex flex-col">
+            <section className="flex flex-col md:flex-row md:pb-4">
+                <figure className="w-full md:w-1/2">
+                    <img src={article_img_url} alt={title} className="w-full h-auto rounded-md"/>
+                </figure>
+                <summary className="md:pl-4 flex flex-col justify-center">
```

Comment:
Summary is only ever intended to be a child of `details` so its use here is incorrect